index.js

```diff
@@ -108,10 +108,18 @@ module.exports = (helpText, options) => {
 	delete argv._;
 
 	const flags = camelcaseKeys(argv, {exclude: ['--', /^\w$/]});
+	const unnormalizedFlags = camelcaseKeys(argv, {exclude: ['--', /^\w$/]});
+
+	if (options.flags !== undefined) {
+		for (const flag of Object.entries(options.flags)) {
+			delete flags[flag[1].alias];
+		}
+	}
 
 	return {
 		input,
 		flags,
+		unnormalizedFlags,
```

Comment:
I'm still not sold on this name. I think we should try to come up with something shorter.

Reply:
I've got `unsortedFlags`, what do you thing about it  🤷‍♂️ ?

Reply:
Not really better. Let's just go with `unnormalizedFlags` for now.